cw_zano/lib/zano_wallet_api.dart

```diff
@@ -45,18 +46,17 @@ mixin ZanoWalletApi {
   void setPassword(String password) => zano.PlainWallet_resetWalletPassword(hWallet, password);
 
   void closeWallet(int? walletToClose, {bool force = false}) async {
-    printV('close_wallet ${walletToClose ?? hWallet}');
+    printV('close_wallet ${walletToClose ?? hWallet}: $force');
     if (Platform.isWindows || force) {
       final result = await _closeWallet(walletToClose ?? hWallet);
       printV('close_wallet result $result');
       openWalletCache.removeWhere((_, cwr) => cwr.walletId == (walletToClose ?? hWallet));
     }
   }
 
-  bool isInit = false;
+  static bool isInit = false;
```

Comment:
why static?

Reply:
Zano requires init() call only once, not once per wallet